### DISCUSSION TITLE
Benewake TFmini I2C

### DISCUSSION THREAD

**Discussion by djholt** [iNavFlight/inav#9358] [Q&A] [answered]
I'm having a hard time finding a definitive answer to this, so I'm hoping this can be quick:

Does iNav support the Benewake TFmini Plus lidar sensor on I2C, or only UART? I'm trying to combine this sensor with an optical flow sensor, and I've run out of UARTs. Thank you for any answers!

**Comment by DzikuVx** [comment, marked as answer]
Only on UART
